docs/docs/getting-started.md

```diff
@@ -41,9 +41,63 @@ $ npm install --save react react-dom babel-preset-react
 $ webpack
 ```
 
-> Note:
->
-> If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
+Ensure you have a `.babelrc` including the react preset.
```

Comment:
Can the babel config be specified in `webpack.config.js` (in loader config)? I think having that is simpler. People could learn about `.babelrc` later on.